modules/common-workflows/pages/workflow-install-from-raw-image.adoc

```diff
@@ -1,62 +1,61 @@
-Raw image format is flexible and compatible with a wide range of virtualization and deployment tools. This guide provides essential information about these pre-configured images.
+RAW images are pre-built, ready-to-use representations of a running operating system. 
+Instead of being installed through a traditional installer, they are directly copied to the hard disk of the target host.
+RAW image format is flexible and compatible with a wide range of virtualization and deployment tools. 
+
+This guide provides essential information about these pre-configured images.
+
 
 
 == Outcome 
 
 Successful completion of this workflow results in successful installation of {productname} Server.
 
 
-== Step-by-step Preparation Instructions
 
+== Step-by-step Instructions
 
 [WARNING]
 ====
 This workflow is complex and time-consuming.
 Make sure that adequate testing is done before deploying the procedure in live environment.
 ====
 
-.Procedure: Prerequisites
+.Procedure: Prepare the Prerequisites
 [role=procedure]
 . *Download the Image:* Download the architecture-specific [literal]``.raw.xz`` image of SUSE Manager 5.0 Server for your target block size.
-
-
-. *Make Image Accessible:* Copy the image to a web server (preferred), NFS server, or expand the .raw file to add directly as a disk in a virtual environment (if supported).
-
-
-. *Boot Media:* Provide a bootable ISO ({sles} 15 SPX or {sle-micro} 5.5) to access the "Rescue System." Add netsetup=1 on the kernel command line for networking, or configure IP via ifcfg.
-
+. *Make Image Accessible:* Copy the image to a web server (preferred), NFS server, or expand the [literal]``.raw`` file to add directly as a disk in a virtual environment (if supported).
+. *Boot Media:* Provide a bootable ISO ({sles} 15 SPX or {sle-micro} 5.5) to access the "Rescue System." 
+  Add [literal]``netsetup=1`` on the kernel command line for networking, or configure IP via [command]``ifcfg``.
 . *Target Disk Setup:* In VMware, create:
 .. A 40GB root disk
 .. Additional storage for persistent SUSE Manager data
+
 											
 .Procedure: Installation Steps
-										
+[role=procedure]										
 . *Prepare Network and Boot Environment*
-
 .. Create a new instance, attach the root and persistent storage, then attach the {sles} or {sle-micro} ISO as a virtual CD.
-.. Boot the instance to the ISO, selecting the “Rescue System.”
+.. Boot the instance to the ISO, selecting the [literal]``Rescue System``.
 .. Add the following on the command line to enable network configuration.
-
 +
 ----
 netsetup=1
 ----
 +
-. *Configure Network & Identify Devices*
-
+. *Configure Network and Identify Devices*
 .. Complete network setup as prompted and log in as root.
-.. Use fdisk to confirm the root device name
+.. Use [command]``fdisk`` to confirm the root device name.
 +
 ----
 fdisk -l
```

Comment:
This step can be skipped since we have the same information in the `lsblk` output when we omit device name.